iree/compiler/Conversion/LinalgToSPIRV/ConvertToSPIRVPass.cpp

```diff
@@ -45,6 +47,29 @@
 #include "mlir/Support/LogicalResult.h"
 #include "mlir/Transforms/DialectConversion.h"
 
+namespace llvm {
+template <>
+struct DenseMapInfo<mlir::iree_compiler::IREE::PlaceholderOp> {
+  static mlir::iree_compiler::IREE::PlaceholderOp getEmptyKey() {
+    auto pointer = llvm::DenseMapInfo<void *>::getEmptyKey();
```

Comment:
nit: no need for `llvm::` since we already wrap this function within llvm namespace.